docs/MANUAL.md

```diff
@@ -327,12 +327,15 @@ Het toevoegen, wijzigen of verwijderen van kanalen gaat op een gelijksoortige ma
 
 Let op: als je KISS gebruikt in combinatie met een bronregister, waarin ook kanalen aanwezig zijn (geconfigureerd of een vaste lijst kanalen), let dan goed op of het van belang is dat de schrijfwijze van een kanaal in KISS exact moet overeenkomen met kanaal in het bronregister. In dat geval moet je ook letten op hoofdletters, kleine, spaties en streepjes.
 
+## Vacs
+Op de pagina 'Vacs' kan je Vraag-Antwoord-Combinaties beheren die zijn opgeslagen in het bij de KISS-installatie geconfigureerde Objectenregister. De lijst op de Vac-overzichtspagina toont alle Vacs met status `actief`. Door middel van de toetsenbord combinatie `Ctrl + f` kun je in de lijst zoeken naar de vraag die bij de Vraag-Antwoord-Combinatie hoort.

```

Comment:
De zin "De lijst op de Vac-overzichtspagina toont alle Vacs met status `actief`." 
klopt niet. 

Heb net getest door de status van een VAC in object te wijzigen naar `"status": "concept",` en die blijft gewoon in de lijst staan. Dus "met status `actief`" mag weg.

Reply:
gedaan